tests/behat/features/bootstrap/Page/SearchPage.php

```diff
@@ -0,0 +1,1402 @@
+<?php
+
+/**
+ * @file
+ * Implements searching functionality
+ */
+
+namespace Page;
+
+use Behat\Mink\Session;
+use SensioLabs\Behat\PageObjectExtension\PageObject\Factory;
+use Stack;
+
+/**
+ * Class SearchPage
+ *
+ * @package Page
+ */
+class SearchPage extends PageBase {
+
+  /**
+   * Stack holding the number of result per search. Primarily used for testing facets.
+   *
+   * @var Stack $expectedResultsCount
+   */
+  private $expectedResultsCount;
+
+  /**
+   * If 0 it will be interpreted as 'all'-
+   *
+   * @var integer $maxPageTraversals
+   */
+  protected $maxPageTraversals = 0;
+
+  /**
+   * Contains all log messages from verbose mode.
+   *
+   * @var string $messageCollection
+   */
+  protected $messageCollection = '';
+
+  /**
+   * Path to search result pages.
+   *
+   * @var string $path
+   */
+
+  protected $path = '/search/ting/{string}';
+
+  /**
+   * Holds the latest search result we scraped from the pages.
+   *
+   * @var array $searchResults
+   */
+  protected $searchResults = array();
+
+
+  /**
+   * Flag for verbose settings.
+   *
+   * @var string $verboseSearchResults
+   */
+  protected $verboseSearchResults = 'off';
+
+  /**
+   * Version 4: Checks if the pagination elements are shown correctly on page.
+   *
+   * First page:   (1) 2 Næste
+   * Second :      forrige 1 (2) 3 Næste
+   * Third:        første forrige 1 2 (3) 4 Næste
+   * Fourth:       første forrige 1 2 3 (4) 5 Næste
+   * etc...
+   * Last page:    første forrige (...) 9 10 11 12 13 14 15 16 (17)   (so næste is not shown)
+   *               første forrige (...) 4 5 6 7 8 9 10 (11) 12 næste
+   * so always show the next page + Næste, and
+   *           all previously shown pages (up to 7 previous, which means ellipsis is shown from ?page=8
+   *
+   * returns a text string containing any deviances found. It doesn't fail on its own. If the returned
+   * string is empty, there was no problems found.
+   */
```

Comment:
@return missing

Reply:
Added and fixed the phpdoc comment altogether.